### DISCUSSION TITLE
Search by genre

### DISCUSSION THREAD

**Discussion by deNoi5e** [sigma67/ytmusicapi#452] [Ideas]
Good afternoon. I apologize if I missed it, but is there an option to search by music genre? If it is not available now, are there any plans to introduce it in the future?

**Comment by deNoi5e** [comment]
Overall, the list of genres doesn't look very extensive :( .
https://gist.github.com/stpe/2951130dfc8f1d0d1a2ad736bef3b703

**Comment by sigma67** [comment]
No idea, you can't search by genre on the web UI can you?

You can do this and go from there: https://ytmusicapi.readthedocs.io/en/stable/reference.html?highlight=genre#ytmusicapi.YTMusic.get_mood_categories
